openedx/core/djangoapps/discussions/docs/decisions/0004-in-context-discussions-linking.rst

```diff
@@ -0,0 +1,134 @@
+Expose Discussion Configuration via HTTP API
+============================================
+
+
+Status
+------
+
+Proposal
+
+
+Context
+-------
+
+As part of the BD-38 initiative (Blended Development, Project 38),
+we have are investigating a new way to set up discussions for a course
+that tries to simplify the way discussions are configured and ties them
+more closely to the course structure.
+
+The first simplification that the new system makes is to remove the
+discussions XBlock, and instead supports simply marking individual units
+as discussable, in which case a discussion UI will show up in-context.
+
+Currently this will only be supported for the edX Forums internal
+discussion provider, but it could be extended in the future to support
+other providers.
+
+In addition to providing a way to mark Units as discussable, there will
+also be a way to simply enable discussion for all units in the course.
+One further control provided here will be to toggle discussions on or
+off for graded discussions.
+
+Finally, it will also be possible to group discussions at a Sequence
```

Comment:
I think you've misunderstood the context here, we're not talking about having or not having group discussions, all of these are group discussions in a way, we're talking about being able to group discussion threads/topics at a sequence level instead of the unit level.